assets/src/application/ui/layout/descriptionList/DescriptionList.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { DescriptionListProps } from './types';
+import './styles.scss';
+
+const DescriptionList: React.FC<DescriptionListProps> = ({ dataSource }) => {
```

Comment:
I've used the same prop as in antd `List`: https://github.com/ant-design/ant-design/blob/master/components/list/index.tsx#L39